src/golang/lib/repos/sqlite/workflow.go

```diff
@@ -45,6 +45,18 @@ func (*workflowReader) Get(ctx context.Context, ID uuid.UUID, DB database.Databa
 	return getWorkflow(ctx, DB, query, args...)
 }
 
+func (*workflowReader) GetByDAG(ctx context.Context, dagID uuid.UUID, DB database.Database) (*models.Workflow, error) {
```

Comment:
This seems to be implemented in this PR. Why is this not implemented in the implementation PR? Did we just miss it?

Reply:
Yup it was missed. For these adhoc changes I felt it was easiest to include it in the PRs as I was doing the refactor, otherwise it creates a lot of fragmented changes.